samples/nametranslation.d.ts.scala

```diff
@@ -16,7 +16,7 @@ object Nametranslation extends js.Object {
   val int8Array: js.typedarray.Int8Array = js.native
   val int16Array: js.typedarray.Int16Array = js.native
   val int32Array: js.typedarray.Int32Array = js.native
-  val readonlyArray: js.Array[Boolean] = js.native
+  val readonlyArray: js.Array[_ <: scala.Boolean] = js.native
```

Comment:
I suggest adding a case in [`Printer.print()` after `: TypeRef`](https://github.com/sjrd/scala-js-ts-importer/blob/5329ef0a82dc91497dd5d0071c0cc95c14388077/src/main/scala/org/scalajs/tools/tsimporter/sc/Printer.scala#L232) for `: Wildcard`, which would use the `p""` interpolator as well (the same way that `printTypeRef` does) so that the `scala.` prefix can be eliminated here.

It's actually quite important for correctness if the bound is one of the special types such as `TypeRef.Union`, because then it's not just about eliding the prefix.